### DISCUSSION TITLE
Linked List

### DISCUSSION THREAD

**Discussion by ankit-akash** [ankit-akash/DSA#16] [Q&A] [answered]
Linked List is classified into 4 types. Name them.

**Comment by Preetiraj3697** [comment]
A linked list is a linear data structure, in which the elements are not stored at contiguous memory locations. The elements in a linked list are linked using pointers.
There are four key types of linked lists:
Singly linked lists
Doubly linked lists
Circular linked lists
Circular doubly linked lists

**Comment by alperencubuk** [comment, marked as answer]
1) Singly linked lists
2) Doubly linked lists
3) Circular linked lists
4) Circular doubly linked lists

**Reply by ankit-akash** [reply]
congrats!🎉 You have achieved the Galaxy Brain badge.
